src/components/content-types/ContentList.tsx

```diff
@@ -214,14 +195,40 @@ export default function ContentList({ documentId }: ContentProps) {
               onDrop={(evt) => onDrop(evt)}
               key={url}
             >
-              <ActionListItem
-                contentUrl={url}
-                defaultAction={actions[0]}
-                actions={actions}
+              <ListMenuItem
+                onClick={() => selectContent(url)}
                 selected={url === currentContent}
               >
-                <Content context="list" url={url} editable={true} />
-              </ActionListItem>
+                <Content
+                  context="list"
+                  url={url}
+                  editable={contentUrlBeingEdited === url} // todo: this should be done in a more principled way
+                  onBlur={() => setContentUrlBeingEdited(undefined)}
```

Comment:
So... let's talk about `onBlur`, probably in realtime with you me and Geoffrey. My instinct is that we should strongly avoid making the interface wider. There are a lot of incidental changes in the patch so I haven't internalized it all yet but I wonder how essential it is.